packages/expo-router/src/fork/getStateFromPath.ts

```diff
@@ -49,7 +47,17 @@ export function getUrlWithReactNavigationConcessions(
   path: string,
   basePath: string | undefined = Constants.expoConfig?.experiments?.basePath
 ) {
-  const parsed = new URL(path, 'https://acme.com');
+  let parsed: URL;
+  try {
+    parsed = new URL(path, 'https://acme.com');
```

Comment:
Really prefer not to use a valid domain name that could point to someone's website. Better to use the .example TLD:

```suggestion
    parsed = new URL(path, 'https://phony.example');
```

"phony" chosen since phony is already a concept for Makefile targets where a name is needed for something that doesn't correspond to a real resource